### PR TITLE
254-RN-Login-page

### PR DESCRIPTION
**Pull request by Horizon19911**

### CODE REVIEW

File: src/components/Login/login.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                <Text style={styles.brandText}>Devx Learning</Text>
+            </View>
+            <Formik
+                validationSchema={LoginSchema}
+                initialValues={{email: '', password: ''}}
+                onSubmit={(values) => {
+                    console.log(values);
+                    setForm({...values});
+                }}>
+                {(FormikProps) => {
```
Comment (minkhant285) [inline src/components/Login/login.tsx]: @Horizon19911 (FormikProps) to (FormikProps:any)

File: src/components/Login/login.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    });
+    return (
+        <View style={styles.body}>
+            <View style={styles.brandField}>
+                <Text style={styles.brandText}>Devx Learning</Text>
+            </View>
+            <Formik
+                validationSchema={LoginSchema}
+                initialValues={{email: '', password: ''}}
+                onSubmit={(values) => {
```
Comment (minkhant285) [inline src/components/Login/login.tsx]: change values to  values: FormValues

File: src/components/Home/home.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                 Devx Learning
+            </Text>
+            <Text style={styles.home_text}>
+                Home
+            </Text>
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
```
Comment (minkhant285) [inline src/components/Home/home.tsx]: @Horizon19911  seperate style file and component file

File: src/components/Login/login.tsx [hunk 4]
```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {useState} from 'react';
+import {styles} from './ultis/styles'
+import {
+    View,
+    Text,
+    TextInput,
+    TouchableOpacity,
+} from 'react-native';
+import {compose} from 'recompose';
```
Comment (minkhant285) [inline src/components/Login/login.tsx]: make format {style} to { style } and change other similar code

**Comment by Horizon19911** [comment]
@oaksoe @minkhant285

**Review by minkhant285 — CHANGES_REQUESTED** [review]
@Horizon19911  I have some PR comments for you.

**Review by minkhant285 — APPROVED** [review]

**Pull request merged**